### PR TITLE
Feat: new delegation modal

### PR DESCRIPTION
**Pull request by JoaquinBattilana**
## General Changes

Design update for governance delegation modal

delegationModal

## Developer Notes

Adds one small update to zero LTV warning. If an asset was never enabled as collateral, the user may still have usageAsCollateral set to true and the warning would be displayed even though the restriction doesn't apply.

---

## Author Checklist

Please ensure you, the author, have gone through this checklist to ensure there is an efficient workflow for the reviewers.

- [ ]  The base branch is set to `main`
- [ ]  The title is using [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) formatting
- [ ]  The Github issue has been linked to the PR in the Development section
- [ ]  The General Changes section has been filled out
- [ ]  Developer Notes have been added (optional)

**If the PR is ready for review:**

- [ ]  The PR is in `Open` state and not in `Draft` mode
- [ ]  The `Ready for Dev Review` label has been added

## Reviewer Checklist

Please ensure you, as the reviewer(s), have gone through this checklist to ensure that the code changes are ready to ship safely and to help mitigate any downstream issues that may occur.

- [ ]  End-to-end tests are passing without any errors
- [ ]  Code style generally follows existing patterns
- [ ]  Code changes do not significantly increase the application bundle size
- [ ]  If there are new 3rd-party packages, they do not introduce potential security threats
- [ ]  If there are new environment variables being added, they have been added to the `.env.example` file as well as the pertinant `.github/actions/*` files
- [ ]  There are no CI changes, or they have been approved by the DevOps and Engineering team(s)
- [ ]  Code changes have been quality checked in the ephemeral URL
- [ ]  QA verification has been completed
- [ ]  There are two or more approvals from the core team
- [ ]  Squash and merge has been checked

### CODE REVIEW

**Comment by JoaquinBattilana** [comment]
Link T-6332

**Comment by iamanastasia** [comment]
@MareskoY Are you saying that we should rename CTA buttons for zero state and active states? Like this?
Screenshot 2023-03-01 at 20 57 42

**Comment by iamanastasia** [comment]
@JoaquinBattilana Could you please change tooltip text here:

Screenshot 2023-03-02 at 22 59 39


Suggested text:

1. Your voting power is based on your AAVE/stkAAVE balance and received delegations. Use it to vote for or against active proposals.

2. Your proposition power is based on your AAVE/stkAAVE balance and received delegations. 
To submit a proposal for minor changes to the protocol, you'll need at least 80.00K power. If you want to change the core code base, you'll need 320k power. Learn more (Link: https://docs.aave.com/governance/aips).


Everything else works like magic for me

File: src/modules/governance/DelegatedInfoPanel.tsx [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+        )}
+      </Box>
+    </Box>
+  );
+};
+
+export const DelegatedInfoPanel = () => {
+  const powers = useVotingPower();
+  const {
+    daveTokens: { aave, stkAave },
```
Comment (grothem) [inline src/modules/governance/DelegatedInfoPanel.tsx:110]: I think this is just an existing typo, but we should probably update `daveTokens` to be `aaveTokens`

File: src/components/transactions/GovDelegation/GovDelegationModalContent.tsx [hunk 2]
```diff
@@ -34,32 +36,48 @@ export enum ErrorType {
   NOT_AN_ADDRESS,
 }
 
-export const GovDelegationModalContent = () => {
-  const { chainId: connectedChainId, readOnlyModeAddress } = useWeb3Context();
+type GovDelegationModalContentProps = {
+  type: ModalType.RevokeGovDelegation | ModalType.GovDelegation;
+};
+
+export const GovDelegationModalContent: React.FC<GovDelegationModalContentProps> = ({ type }) => {
```
Comment (grothem) [inline src/components/transactions/GovDelegation/GovDelegationModalContent.tsx:43]: I'm not sure why, but there's some styling differences for this component when shown for revoke and delegation

Comment (grothem) [reply src/components/transactions/GovDelegation/GovDelegationModalContent.tsx:43]: Oh I see, it's when some of the toggles are disabled, the styling is different. I think @MareskoY comment makes sense, to not show the selector if only one option is going to be enabled.

File: src/components/transactions/GovDelegation/GovDelegationModalContent.tsx [hunk 3]
```diff
@@ -97,42 +119,83 @@ export const GovDelegationModalContent = () => {
...
+        <>
+          <Typography variant="description" color="text.secondary" mb={1}>
+            <Trans>Recipient address</Trans>
+          </Typography>
+          <FormControl
+            error={delegateAddressBlockingError !== undefined}
+            variant="standard"
+            fullWidth
+          >
+            <TextField
```
Comment (grothem) [inline src/components/transactions/GovDelegation/GovDelegationModalContent.tsx:187]: We should look at making a component for the input that looks up an address from an ENS name or UD (for watch mode) and use that here.

**Review by defispartan — APPROVED** [review]

**Review by defispartan — DISMISSED** [review]

**Comment by MareskoY** [comment]
Link T-7862

**Comment by MareskoY** [comment]
Link T-7862

**Comment by MareskoY** [comment]
> @MareskoY Are you saying that we should rename CTA buttons for zero state and active states? Like this? Screenshot 2023-03-01 at 20 57 42

I asked about changing text for the modal button 

https://github.com/aave/interface/pull/1499#issuecomment-1450281916

**Review by defispartan — DISMISSED** [review]
Looks great!

**Review by defispartan — APPROVED** [review]

**Review by defispartan — APPROVED** [review]

**Review by MareskoY — DISMISSED** [review]

**Review by defispartan — APPROVED** [review]

**Review by MareskoY — APPROVED** [review]

**Pull request merged**